src/Application/Game.cpp

```diff
@@ -1102,7 +1102,7 @@ void Game::EventLoop() {
                     } else {
                         pParty->field_6E4_set0_unused = 0;
                         pParty->field_6E0_set0_unused = 0;
-                        CastSpellInfoHelpers::Cancel_Spell_Cast_In_Progress();
+                        CastSpellInfoHelpers::CancelSpellCastInProgress();
```

Comment:
Since we're renaming the functions, let's rename them according to the codestyle: `cancelSpellCastInProgress`.